draft-ietf-rats-corim.md

```diff
@@ -1882,6 +1900,13 @@ The Evidence ECT fields are populated as described in {{sec-phase1-trans}} and {
 Evidence transformation algorithms may be well-known, may be defined by a CoRIM profile ({{sec-corim-profile-types}}), or may be supplied dynamically.
 The handling of dynamic Evidence transformation algorithms is out of scope for this document.
 
+### Appraisal hermeticity
+
+The Appraisal Context at the and of Phase 1 constitutes all inputs to the Appraisal Procedure.
+The same Appraisal Context processed at any time MUST produce the same Attestation Results.
+
+The reason to lock the inputs before Attestation Appraisal is to ensure consistent ACS construction when accounting for any profile-driven semantics.
```

Comment:
Please help wordsmith this. There are obviously different levels of hermeticity and auditability that Verifier policy will impose. I don't want to require absolute determinism since that imposes a requirement on policy evaluation.
I don't like "same" since we're not prescribing any Attestation Result construction or representation, so there is no strong notion of sameness.

If we go with 
```suggestion
The reason to lock the inputs before Attestation Appraisal is for all Appraisal Procedure dependencies to be accounted for before interpreting them.                                                                                            
Fully determined inputs SHOULD ensure consistent ACS construction when accounting for any profile-driven semantics.                                                                                                                             
```

Reply:
I agree that the old L1906 is not 100% correct: it should probably have said instead:

> The same Appraisal Context processed at any time MUST produce the same ACS.

I like the text you suggest.  The only thing is we should state more strongly that what we are trying to achieve is _deterministic ACS construction._

Reply:
> Fully determined inputs SHOULD ensure consistent ACS construction when accounting for any profile-driven semantics.

```suggestion
Given the same Appraisal Context, different Verifier appraisals MUST produce deterministic results.

Verifier inputs are locked (see Step 1) before Attestation Appraisal to ensure ACS construction is deterministic.
```
Text above addresses the question of profile or policy inputs rendering non-deterministic ACS results.